### ISSUE TITLE
Error with compound binding containing 2 property bindings

### ISSUE THREAD

**Issue by MeTaNoV** [PolymerLabs/polylint#113]
Hi,
given the following declaration:
`<a data-route="game" href$="{{_gameBaseUrl}}/{{game.__firebaseKey__}}">`
polylint detects a false error like this:
`Property '_gameBaseUrl}}/{{game' bound to attribute 'href$' not found in 'properties' for element 'bgo-game-item'`
I can take a look for a PR if needed.
Cheers,
Pascal

**Comment by aendrew** [comment]
This also happens outside of computed attributes, i.e., if you do something like:

`<span>[[aBinding]]: Oh look, here's [[anotherBinding]]</span>`

**Comment by mvolkmann** [comment]
Yeah, the issue is having more than one binding on the same line.
